perun-base/src/main/java/cz/metacentrum/perun/audit/events/AttributesManagerEvents/AttributePolicyCollectionSet.java

```diff
@@ -0,0 +1,38 @@
+package cz.metacentrum.perun.audit.events.AttributesManagerEvents;
+
+import cz.metacentrum.perun.audit.events.AuditEvent;
+import cz.metacentrum.perun.audit.events.EngineIgnoreEvent;
+import cz.metacentrum.perun.core.api.AttributePolicyCollection;
+import cz.metacentrum.perun.core.api.AttributeRights;
+
+public class AttributePolicyCollectionSet extends AuditEvent implements EngineIgnoreEvent {
+
+	private AttributePolicyCollection policyCollection;
+	private int attributeId;
```

Comment:
This might be unnecessary field because `attributeId` is already part of `policyCollection`. What do you think?